frontend/src/hooks/useGetBoardInfo.ts

```diff
@@ -1,45 +1,47 @@
 import { BOARD_CONFIG_INFO_ERROR, BOARD_CONFIG_INFO_TITLE } from '@src/constants/resources';
 import { boardInfoClient } from '@src/clients/board/BoardInfoClient';
-import { BoardInfoRequestDTO } from '@src/clients/board/dto/request';
+import { BoardInfoConfigDTO } from '@src/clients/board/dto/request';
 import { AXIOS_REQUEST_ERROR_CODE } from '@src/constants/resources';
 import { AxiosResponse, HttpStatusCode } from 'axios';
 import { ReactNode, useState } from 'react';
 import get from 'lodash/get';
+import dayjs from 'dayjs';
 
 export type JiraColumns = Record<string, string>[];
 export type TargetFields = Record<string, string>[];
 export type Users = string[];
 export interface BoardInfoResponse {
   jiraColumns: JiraColumns;
   targetFields: TargetFields;
+  ignoredTargetFields: TargetFields;
   users: Users;
 }
 export interface useGetBoardInfoInterface {
-  getBoardInfo: (data: BoardInfoRequestDTO) => Promise<AxiosResponse<BoardInfoResponse>>;
+  getBoardInfo: (data: BoardInfoConfigDTO) => Promise<Awaited<AxiosResponse<BoardInfoResponse>[]> | undefined>;
   isLoading: boolean;
   errorMessage: Record<string, ReactNode>;
 }
 
 const codeMapping = (code: string | number) => {
   const codes = {
     [HttpStatusCode.BadRequest]: {
-      title: BOARD_CONFIG_INFO_TITLE.INVALID_INPUT,
-      message: BOARD_CONFIG_INFO_ERROR.NOT_FOUND,
+      title: BOARD_CONFIG_INFO_TITLE.GENERAL_ERROR,
+      message: BOARD_CONFIG_INFO_ERROR.GENERAL_ERROR,
       code: HttpStatusCode.BadRequest,
     },
     [HttpStatusCode.Unauthorized]: {
-      title: BOARD_CONFIG_INFO_TITLE.UNAUTHORIZED_REQUEST,
-      message: BOARD_CONFIG_INFO_ERROR.NOT_FOUND,
+      title: BOARD_CONFIG_INFO_TITLE.GENERAL_ERROR,
+      message: BOARD_CONFIG_INFO_ERROR.GENERAL_ERROR,
       code: HttpStatusCode.Unauthorized,
     },
     [HttpStatusCode.Forbidden]: {
-      title: BOARD_CONFIG_INFO_TITLE.FORBIDDEN_REQUEST,
-      message: BOARD_CONFIG_INFO_ERROR.FORBIDDEN,
+      title: BOARD_CONFIG_INFO_TITLE.GENERAL_ERROR,
+      message: BOARD_CONFIG_INFO_ERROR.GENERAL_ERROR,
       code: HttpStatusCode.Forbidden,
     },
     [HttpStatusCode.NotFound]: {
-      title: BOARD_CONFIG_INFO_TITLE.NOT_FOUND,
-      message: BOARD_CONFIG_INFO_ERROR.NOT_FOUND,
+      title: BOARD_CONFIG_INFO_TITLE.GENERAL_ERROR,
+      message: BOARD_CONFIG_INFO_ERROR.GENERAL_ERROR,
```

Comment:
are u sure all of the above errors will be mapping to `GENERAL_ERROR`?

If yes, duplicated code

Reply:
Because the future code will refactor this code mapping, we choose to maintain this style.